Extensions/Content/Dnn.PersonaBar.Pages/Services/PagesController.cs

```diff
@@ -303,7 +303,7 @@ public HttpResponseMessage DeletePage(PageItem page)
 
             try
             {
-                _pagesController.DeletePage(page);
+                _pagesController.DeletePage(page, PortalSettings, hardDelete);
```

Comment:
hm, the code before was not passing in `PortalSettings`. will it be the same object?

Reply:
Yes, the both call to current portal settings.